src/pineko/check.py

```diff
@@ -53,13 +53,20 @@ def check_grid_and_eko_compatible(pineappl_grid, operators, xif):
         operators
     xif : float
         factorization scale variation
+    max_as: int
+        max order of alpa_s
+    max_al: int
+        max order of alpha
 
     Raises
     ------
     ValueError
         If the operators and the grid are not compatible.
     """
-    x_grid, _pids, _mur2_grid, muf2_grid = pineappl_grid.axes()
+    order_mask = pineappl.grid.Order.create_mask(pineappl_grid.orders(), max_as, max_al)
```

Comment:
https://github.com/NNPDF/pineko/blob/2cbedfb06f19413cc18d5b86c41170287267a1da/src/pineko/evolve.py#L25

It will be soon as a default inside PineAPPLpy

Reply:
Driooed from where?

Reply:
Yes, and that is more or less what I did in #75. And the reason for that was exactly to avoid that this check depended on max_as and max_al

Reply:
So, if you intend "dropped from the grid", of course the trivial mask would be adaptive, since it depends on `grid.orders()`, so there is nothing that `max_as/l` could cut.

Of course, HO can't be dropped from the EKO, since it is a resummed object and they are not stored separately.

Reply:
Yes, this concerns only the grid.